frontend/src/libs/helpers/sanitize-html/sanitize-html.helper.ts

```diff
@@ -1,5 +1,11 @@
 import sanitizeHtmlInternal from 'sanitize-html';
 
-const sanitizeHtml = (html: string): string => sanitizeHtmlInternal(html);
+const sanitizeHtml = (html: string): string =>
+  sanitizeHtmlInternal(html, {
```

Comment:
I guess it's better to use brackets here
https://github.com/BinaryStudioAcademy/bsa-2023-writorium/pull/84#discussion_r1310608821